src/persist-client/tests/machine/batch

```diff
@@ -56,144 +56,6 @@ fetch-batch input=b0 stats=lower
 ----
 <empty>
 
-# regression: runs after the first one can contain multiple parts.
-# note: target part size set so that each part contains 2 keys.
-write-batch output=b0 lower=0 upper=2 target_size=50
```

Comment:
Deleted these tests because the functionality is gone: we no longer try and discover run boundaries automatically.